src/Stratis.Bitcoin.IntegrationTests/BlockStore/ReorgToLongestChainSteps.cs

```diff
@@ -46,35 +47,31 @@ private void four_miners()
         {
             this.jingNode = this.nodeBuilder.CreateStratisPowNode(this.network);
             this.jingNode.Start();
-            this.jingNode.NotInIBD();
-            this.jingNode.FullNode.WalletManager().CreateWallet(WalletPassword, WalletZero, WalletPassphrase);
+            this.jingNode.NotInIBD().WithWallet();
 
             this.bobNode = this.nodeBuilder.CreateStratisPowNode(this.network);
             this.bobNode.Start();
-            this.bobNode.NotInIBD();
-            this.bobNode.FullNode.WalletManager().CreateWallet(WalletPassword, WalletZero, WalletPassphrase);
+            this.bobNode.NotInIBD().WithWallet();
 
             this.charlieNode = this.nodeBuilder.CreateStratisPowNode(this.network);
             this.charlieNode.Start();
-            this.charlieNode.NotInIBD();
-            this.charlieNode.FullNode.WalletManager().CreateWallet(WalletPassword, WalletZero, WalletPassphrase);
+            this.charlieNode.NotInIBD().WithWallet();
 
             this.daveNode = this.nodeBuilder.CreateStratisPowNode(this.network);
             this.daveNode.Start();
-            this.daveNode.NotInIBD();
-            this.daveNode.FullNode.WalletManager().CreateWallet(WalletPassword, WalletZero, WalletPassphrase);
+            this.daveNode.NotInIBD().WithWallet();
 
-            TestHelper.ConnectAndSync(this.jingNode, this.bobNode);
-            TestHelper.ConnectAndSync(this.bobNode, this.charlieNode);
-            TestHelper.ConnectAndSync(this.charlieNode, this.daveNode);
+            TestHelper.Connect(this.jingNode, this.bobNode);
```

Comment:
Just wondering why it was necessary to change which method is being called

Reply:
same here. Are you sure you're happy for the test to carry on even if the nodes aren't synced?

Reply:
Well at this point the nodes have not point anything yet so it was redundant for the sync when there's nothing to sync ;)